docs/play/cdc-pay-integration.md

```diff
@@ -0,0 +1,194 @@
+    ```csharp
+    using UnityEngine;
+    using UnityEngine.Networking;
+    using System.Collections;
+
+    public class CDCPayment : MonoBehaviour
+    {
+        void Start()
+        {
+            StartCoroutine(Pay());
+        }
+
+        IEnumerator Pay()
+        {
+            WWWForm form = new WWWForm();
+            form.AddField("amount", "1000");
+            form.AddField("currency", "USD");
+            form.AddField("description", "Product Name");
+            form.AddField("return_url", "http://YOUR_WEBSITE.com/orders/123/complete");
+            form.AddField("cancel_url", "http://YOUR_WEBSITE.com/orders/123/fail");
+
+            using (UnityWebRequest www = UnityWebRequest.Post("https://pay.crypto.com/api/payments", form))
+            www.SetRequestHeader("Authorization", "Bearer SECURITY_KEY");
+            {
+                yield return www.SendWebRequest();
+
+                if (www.result != UnityWebRequest.Result.Success)
+                {
+                    Debug.Log(www.error);
+                }
+                else
+                {
+                    Debug.Log(www.downloadHandler.data);
+                }
+            }
+        }
+    }
+    ```
+
+## Unity Webview Module
+This section illustrates how a new webview game object is created to display the CDC Pay page within the game. To install the webview module, please follow the examples below.
+
+1. Create a new 3d Game in Unity.
+
+2. Add the following dependency to the manifest file in `Packages/manifest.json`
+    ```json
+    "net.gree.unity-webview": "https://github.com/gree/unity-webview.git?path=/dist/package",
+    ```
+
+3. Create a button game object.
+    <img src="./assets/web-view/web-view-integration-step-3.png" />
+
+4. Create a script called `CdcPay.cs` and add the following code to your
+
+    ```csharp
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEngine;
+    using UnityEngine.SceneManagement;
+    using UnityEngine.UI;
+    using UnityEngine.Networking; 
+
+    public class CdcPay : MonoBehaviour
+    {
+        public Text status;
+        WebViewObject webViewObject;
+
+        public class Response {
+            public string payment_url;
+        }
+
+        public void OnCDCPay()
+        {
+            StartCoroutine(Checkout());
+        }
+
+        IEnumerator Checkout()
+        {
+            WWWForm form = new WWWForm();
+            form.AddField("amount", "1000");
+            form.AddField("currency", "USD");
+            form.AddField("description", "Product Name");
+            form.AddField("return_url", "http://YOUR_WEBSITE.com/orders/123/complete");
+            form.AddField("cancel_url", "http://YOUR_WEBSITE.com/orders/123/fail");
+
+            using (UnityWebRequest www = UnityWebRequest.Post("https://pay.crypto.com/api/payments", form))
+            {
+                www.SetRequestHeader("Authorization", "Bearer pk_test_Lp6yX1M1fyx4zmyAyzLNwH1H");
```

Comment:
load the key from an environment or name it "YOUR_PUBLISHABLE_KEY"?

Reply:
fixed